VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.spec.ts

```diff
@@ -50,6 +61,19 @@ describe('NavigatorComponent', () => {
         expect(router.navigate).toHaveBeenCalledWith([pageUrls.JudgeHearingList]);
     });
 
+    it('should go to staff member hearing list if user is a StaffMember and staff-member-feature is ON', async () => {
+        featureFlagServiceSpy.getFeatureFlagByName.and.returnValue(of(false));
```

Comment:
Shouldn't this test fail? Returning false means the feature is turned off right?

Reply:
It looks like this should  be in the below test

Reply:
fixed